### PR TITLE
Add Server Variables doc

### PR DESCRIPTION
**Pull request by shirhatti**
Complete Server variables doc

### CODE REVIEW

**Comment by shirhatti** [comment]
@bangbingsyb Can you review? Especially the new CRYPT_* server variables

File: iis/web-dev-reference/server-variables.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+| CRYPT_HASH_ALG_ID     | The hashing algorithm used for generating Message Authentication Codes (MACs). The hex value of the algorithm is established in the [*ALG_ID* data type](https://msdn.microsoft.com/en-us/library/windows/desktop/aa375549(v=vs.85).aspx).<br/>For example, MD5(CALG_MD5) will be `8003`.|
+| CRYPT_KEYEXCHANGE_ALG_ID | The key-exchange algorithm used to establish the connection. The hex value of the algorithm is established in the [*ALG_ID* data type](https://msdn.microsoft.com/en-us/library/windows/desktop/aa375549(v=vs.85).aspx).<br/>For example, Elliptic curve Diffie-Hellman key exchange algorithm(CALG_ECDH) will be `aa05`. |
+| CRYPT_PROTOCOL        | The cryptographic protocol used to establish the connection. The hex value of the protocol is established in the [*SecPkgContext_ConnectionInfo* structure](https://msdn.microsoft.com/en-us/library/windows/desktop/aa379819(v=vs.85).aspx). For an HTTP request, the value will be `-`.<br/>For example, **TLS 1.2** will be `400`. |
```
Comment (bangbingsyb) [inline iis/web-dev-reference/server-variables.md]: Consider to unify the link format: e.g. ALG_ID and SecPkgContext_ConnectionInfo use italic format, while AllowPathInfoForScriptMappings and AllowPathInfoForScriptMappings are not. Not sure what's the convention for our existing docs, but normal font seems to look better to me.

Comment (shirhatti) [inline iis/web-dev-reference/server-variables.md]: Noted.

Comment (shirhatti) [inline iis/web-dev-reference/server-variables.md]: Agreed. I'll drop the italics

**Pull request merged**